app/services/idv/profile_maker.rb

```diff
@@ -48,7 +48,8 @@ def save_profile(
 
     def set_idv_level(in_person_verification_needed:, selfie_check_performed:)
       if in_person_verification_needed
-        if IdentityConfig.store.in_person_proofing_enforce_tmx
+        if IdentityConfig.store.in_person_proofing_enforce_tmx &&
+           FeatureManagement.proofing_device_profiling_decisioning_enabled?
```

Comment:
@matthinz I think this addresses your concerns [here](https://github.com/18F/identity-idp/pull/10371#discussion_r1554239689)

Reply:
You might consider moving this into a new method in `FeatureManagement`--I believe the intention there is that when we have these kind of relationships between config flags we can encapsulate them in method in there. For example, you could do `FeatureManagement.ipp_proofing_device_profiling_decisioning_enabled?` (side note: I just filed a ticket to come up with a better name than "proofing_device_profiling" but for now that is what we have)

Reply:
@matthinz I think we have a bunch of these flags that could probably be moved in to that class. I think I'll open our own ticket for consolidating our features. Thanks!